### ISSUE TITLE
Delete task > can not view bug (that is belong to the deleted task)

### ISSUE THREAD

**Issue by tienvx** [tienvx/mbt-bundle#365]
Need to reload page

**Comment by tienvx** [comment]
First, task not found 404
```
XHRGEThttp://localhost/api/tasks/1
[HTTP/1.1 404 Not Found 129ms]

	
JSON	
@context	/api/contexts/Error
@type	hydra:Error
hydra:title	An error occurred
hydra:description	Not Found
Response payload	

1

{"@context":"\/api\/contexts\/Error","@type":"hydra:Error","hydra:title":"An error occurred","hydra:description":"Not Found"}


```
Then 401 Authentication Required while checking that error

```
XHRGEThttp://localhost/api/contexts/Error
[HTTP/1.1 401 Unauthorized 95ms]

XMLHttpRequest.prototype.send<
http://localhost:3000/:1:781
_get/<
xhr.js:123
_get
xhr.js:112
loader
xhr.js:59
./node_modules/jsonld/lib/util.js/</api.callbackify/<
util.js:488
add
RequestQueue.js:39
./node_modules/jsonld/lib/util.js/</api.callbackify/<
util.js:488
wrapLoader/<
RequestQueue.js:25
./node_modules/jsonld/lib/util.js/</api.callbackify/<
util.js:488
./node_modules/jsonld/lib/util.js/</api.callbackify/<
util.js:488
retrieve/<
context.js:1230
retrieve
context.js:1213
_retrieveContextUrls
context.js:1190
./node_modules/jsonld/lib/context.js/api.getAllContexts
context.js:1123
wrapper/jsonld.expand<
jsonld.js:352
./node_modules/jsonld/lib/util.js/</api.callbackify/<
util.js:488
_default/<
fetchHydra.js:46

```

This is the code that cause the problem: https://github.com/api-platform/admin/blob/master/src/hydra/fetchHydra.js#L32

**Comment by tienvx** [comment]
Fixed by using version [1.0.1](https://github.com/api-platform/admin/releases/tag/v1.0.1) of api platform admin
